packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -25,6 +25,7 @@ export enum ReducerID {
   allIsZero = 'allIsZero',
   allIsNull = 'allIsNull',
   allValues = 'allValues',
+  allUniqueValues = 'allUniqueValues',
```

Comment:
```suggestion
  uniqueValues = 'uniqueValues',
```

I don't think we need the "all" prefix -- it is used with the other boolean checks